src/output.h

```diff
@@ -85,17 +90,50 @@ typedef struct
     sig_component_t component[MAX_SIG_COMPONENTS];
 } sig_service_t;
 
+typedef struct
+{
+    unsigned int seq;
+    unsigned int size;
+    uint8_t data[MAX_PDU_LEN];
+} packet_t;
+
+typedef struct
+{
+    packet_t *ptr;
+
+    unsigned int size, read, write;
+    unsigned int latency, avg, delay;
+
+    unsigned int clock;
+    int pos;
+} elastic_buffer_t;
+
+typedef struct
+{
+    NeAACDecHandle aacdec;
```

Comment:
The fact that this is defined outside `#ifdef HAVE_FAAD2` causes compilation to fail if nrsc5 is built without FAAD2 (`-DUSE_FAAD2=OFF`):

```
In file included from /home/argilo/git/nrsc5/src/input.h:13,
                 from /home/argilo/git/nrsc5/src/acquire.c:21:
/home/argilo/git/nrsc5/src/output.h:113:5: error: unknown type name ‘NeAACDecHandle’
  113 |     NeAACDecHandle aacdec;
      |     ^~~~~~~~~~~~~~
make[2]: *** [src/CMakeFiles/nrsc5_object.dir/build.make:76: src/CMakeFiles/nrsc5_object.dir/acquire.c.o] Error 1
make[1]: *** [CMakeFiles/Makefile2:104: src/CMakeFiles/nrsc5_object.dir/all] Error 2
make: *** [Makefile:136: all] Error 2
```

Reply:
`decode_t` is only used if `HAVE_FAAD2` is defined, so I think we could safely put this whole `typedef` inside `#ifdef HAVE_FAAD2`.

Reply:
I swapped around. `elastic_buffer_t` contains `decoder_t` This seems to make this a lot better. I fixed the compilation.

Reply:
I t4ested this and I can compile with `DUSE_FAAD2=OFF`. Am going to mark this as resolved unless you cannot reproduce this.